BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -38,17 +50,31 @@ enum FeatureFlag: String, Codable {
         ]
     }
 
+    /// The initial values for feature flags.
+    /// If `isRemotelyConfigured` is true for the flag, then this will get overridden by the server;
+    /// but if `isRemotelyConfigured` is false for the flag, then the value here will be used.
+    /// This is a helpful way to manage local feature flags.
+    static let initialValues: [FeatureFlag: AnyCodable] = [
```

Comment:
🤔  I guess we can reduce the number of switch we need to configure all of this:
- `debugMenuFeatureFlags`: we can return directly here all the feature flags which its `raw` value doesn't start with `test-`.
- `name`: we can get the `raw` value and then change `-` for ` ` (space) and then apply `capitalized` so we get the same result as we do today without having to configure each flag on the switch. There's the special case of `enableCipherKeyEncryption` but in this case we can just check if there is no `-` then get the `raw` value and add a space each time we encounter a capital letter and then perform `capitalized` to uppercase the first letter as well.

Doing this we reduce these switches and is easier to maintain.

Reply:
I'm not sure I want to key off of `test` for determining if something goes in the debug menu, to give us the ability to control that a little more firmly in the future (in case there's something we don't want to put in there for some reason, such as cipher key encryption currently, since it's determined by completely different means than this system). But making that just an array of things in there is less cumbersome than a switch, I feel.

Doing something programmatic to get the name seems reasonable, since the only time the name should crop up is in the debug menu anyways.

I don't know that we can get rid of the `isRemotelyConfigured` per se, though we *could* convert it into an array of the flags that *are* remotely configured that can then be checked, which would reduce a switch.

I'm still fiddling around with the "struct to hold all this" idea to see how I like it (and discovering a bug in this code in the process 🙃 ), though. Definite tradeoffs, and I think ultimately it's not the better way to go. I'd rather move in the direction of having arrays of flags with certain properties, and we just check those—it does a better job of answering the question "what all flags have this property", which I think is a more useful question when looking at code than "what properties does this flag have", though I can see an argument either way. I'm still playing with it.

Reply:
IMO the debug menu stuff should be something not to spend much time to maintain, something simple with a bit of customization; additionally I believe that 90% of flags we'll introduce will be on that debug menu and not determined by some particular logic as cipher key encryption. 
Moreover, most feature flags are not supposed to live forever so I'd like to reduce the number of places we need to think of when we add/remove such flags.
Furthermore, about leaving the thing there as is / or as an array for further customization in the future in case we need it I'd just follow YAGNI and if in the future we encounter some weird flag with custom logic we need to add then we'll add it => I'd just key off `test-` and explicitly remove the cipher key encryption flag from the `debugMenuFeatureFlags`. I think with this we cover most cases therefore less effort to maintain as with array where we'd need to add/remove the key from the array every time a flag changes.

Reply:
Makes sense, and point taken on the debug menu. I can slim that down a fair bit for sure. I keep forgetting feature flags are meant to be transient 😂 

So 👍🏻 , there'll be some more commits coming soon for this.

Reply:
heheh great thanks Katherine!